packages/visx-visx/src/index.ts

```diff
@@ -1,33 +1,61 @@
-// has name conflicts with other packages
-import * as XYChartPackage from '@visx/xychart';
+import * as XYChart from '@visx/xychart';
+import * as Annotation from '@visx/annotation';
+import * as Axis from '@visx/axis';
+import * as Bounds from '@visx/bounds';
+import * as Brush from '@visx/brush';
+import * as ClipPath from '@visx/clip-path';
+import * as Curve from '@visx/curve';
+import * as Drag from '@visx/drag';
+import * as Event from '@visx/event';
+import * as Geo from '@visx/geo';
+import * as Glyph from '@visx/glyph';
+import * as Gradient from '@visx/gradient';
+import * as Grid from '@visx/grid';
+import * as Group from '@visx/group';
+import * as Heatmap from '@visx/heatmap';
+import * as Hierarchy from '@visx/hierarchy';
+import * as Legend from '@visx/legend';
+import * as Marker from '@visx/marker';
+import * as MockData from '@visx/mock-data';
+import * as Network from '@visx/network';
+import * as Pattern from '@visx/pattern';
+import * as Point from '@visx/point';
+import * as Responsive from '@visx/responsive';
+import * as Scale from '@visx/scale';
+import * as Shape from '@visx/shape';
+import * as Text from '@visx/text';
+import * as Tooltip from '@visx/tooltip';
+import * as Voronoi from '@visx/voronoi';
+import * as Zoom from '@visx/zoom';
 
-export * from '@visx/annotation';
-export * from '@visx/axis';
-export * from '@visx/bounds';
-export * from '@visx/brush';
-export * from '@visx/clip-path';
-export * from '@visx/curve';
-export * from '@visx/drag';
-export * from '@visx/event';
-export * from '@visx/geo';
-export * from '@visx/glyph';
-export * from '@visx/gradient';
-export * from '@visx/grid';
-export * from '@visx/group';
-export * from '@visx/heatmap';
-export * from '@visx/hierarchy';
-export * from '@visx/legend';
-export * from '@visx/marker';
-export * from '@visx/mock-data';
-export * from '@visx/network';
-export * from '@visx/pattern';
-export * from '@visx/point';
-export * from '@visx/responsive';
-export * from '@visx/scale';
-export * from '@visx/shape';
-export * from '@visx/text';
-export * from '@visx/tooltip';
-export * from '@visx/voronoi';
-export * from '@visx/zoom';
-
-export const XYChart = XYChartPackage;
+export default {
```

Comment:
```suggestion
export {
```

I think we _don't_ want this to be the `default` export because it would prevent this style of importing:

```ts
// error
import { Shape } from '@visx/visx';

// would have to do
import visx from '@visx/visx';
const { Shape } = visx;
```

In the test file then you would need to fallback to importing everything via

```ts
import * as visx from '...';
```